configure.ac

```diff
@@ -392,7 +392,10 @@ AC_ARG_ENABLE([oldvpn_commands],
   AS_HELP_STRING([--enable-oldvpn-commands], [Keep old vpn commands]))
 AC_ARG_ENABLE(rpki,
   AS_HELP_STRING([--enable-rpki], [enable RPKI prefix validation support]))
+AC_ARG_ENABLE([clippy-only],
+  AS_HELP_STRING([--enable-clippy-only], [Only build clippy]))
 
+AS_IF([test "${enable_clippy_only}" != "yes"], [
 AC_CHECK_HEADERS(json-c/json.h)
```

Comment:
What does json-c have to do with clippy only?

Reply:
I don't want to have unnecessary dev libs installed on the host, almost anything should be used from the build environment

Reply:
if I understand the config will check everything if clippy_only is not enabled, meaning normal build